interface/src/app/utils.js

```diff
@@ -0,0 +1,41 @@
+angular.module('adage.utils', [
+
+])
+
+.factory('MathFuncts', [function() {
+  return {
+    /*
+     * The binomLog and hyperGeometricTest functions are based heavily on the
+     * 'binomlog' and 'hyperg' functions in /integrator/static/js/imp.js
+     * See code here:
+     * https://bitbucket.org/cgreene/integrator/src/1194f7ae1b11057a4649114e1f55570542acbcde/integrator/static/js/imp.js?at=giant&fileviewer=file-view-default
+    */
+    binomLog: function(n, k) {
```

Comment:
I wonder whether it is a good idea to put utils.js and utils.spec.js into a sub-directory such as **app/utils/**. I expect that we will write more and more utility code. If we dump everything into this util.js, it will become very big eventually, and we have to break it up eventually.